ouroboros-consensus/docs/report/chapters/future/genesis.tex

```diff
@@ -60,18 +60,17 @@ \subsection{Background: understanding the Longest Chain rule}
 \end{center}
 %
 This motivates the first part of the Longest Chain rule: chain length is a
-useful proxy for chain density, which in turn reflects honesty.
-The second part of the rule---the intersection
-point is no more than $k$ blocks away from our tip---is important because we can
-only meaningfully compare density \emph{near the intersection point}. As we get
-further away from the intersection point, an adversary can start to influence
-the leadership schedule. This means that if the adversary's chain forks off from
-the honest chain far back enough, they can construct a chain that is longer than
-the honest chain. The Longest Chain  rule therefore restricts rollback, so that
-we will simply not even consider chains that fork off that far back. We can
-still resolve minor forks that happen in the honest chain during the normal
-operation of the protocol, because---so the analysis guarantees---those will not
-be deeper than $k$ blocks.
+useful proxy for chain density, which in turn reflects honesty. The second part
+of the rule---the intersection point is no more than $k$ blocks away from our
+tip---is important because we can only meaningfully compare density \emph{near
+  the intersection point}. As we get further away from the intersection point,
+an adversary can start to influence the leadership schedule. This means that if
+the adversary's chain forks off from the honest chain far back enough, they can
+construct a chain that is longer than the honest chain. The Longest Chain rule
+therefore restricts rollback, so that we will simply not even consider chains
+that fork off that far back. We can still resolve minor forks that happen in the
+honest chain during the normal operation of the protocol, because---so the
+analysis guarantees---those will not be deeper than $k$ blocks.
```

Comment:
Huh, I just tried to wrap this paragraph to see whether our Emacs config is equal and the paragraph changed! Specifically, the line ending with "near" ends with "near the" after rewrapping locally. I'm using Emacs 27.1, plain `latex-mode`, no AUCTeX. You?

Unfortunate, but no big deal. I guess the last one to touch it, wins :slightly_smiling_face:.

For the other two paragraphs below, rewrapping was idempotent.

Reply:
GNU Emacs 26.1 (build 2, x86_64-pc-linux-gnu, GTK+ Version 3.24.5) of 2019-09-22, modified by Debian

Whatever LaTeX mode is default; I don't recall configuring anything.

Reply:
At least it's a different version, so that might explain it :relieved:.

> [..] no big deal. I guess the last one to touch it, wins

So I'm going to merge it.